src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/FontCache/FamilyCollection.cs

```diff
@@ -474,7 +474,11 @@ private static Text.TextInterface.Font GetFontFromFamily(Text.TextInterface.Font
                 // and at every locale name this font face has.
                 foreach (KeyValuePair<CultureInfo, string> name in font.FaceNames)
                 {
+#if NET
                     if (faceName.Equals(name.Value, StringComparison.OrdinalIgnoreCase))
+#else
+                    if (faceName == name.Value.AsSpan())
```

Comment:
This changed the comparison from OrdinalIgnoreCase -> Ordinal, and likely a bug.